<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newdialog</class>
 <widget class="QDialog" name="newdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>332</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>332</width>
    <height>450</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Nowe zlecenie</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/img/img/logo2.png</normaloff>:/img/img/logo2.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox{
font: 10pt &quot;Calibri&quot;;
color: rgb(57, 57, 57);
border-radius:12px;
background-color: rgb(160, 160, 160);
}

QDialog{
background-color: rgb(57, 57, 57);
font: 10pt &quot;Calibri&quot;;
}

QLabel{
font: 10pt &quot;Calibri&quot;;
color: rgb(57, 57, 57);
}

QLineEdit{
font: 75 10pt &quot;Calibri&quot;;
color: rgb(57, 57, 57);
background-color: rgb(246, 246, 246);
border-style: groove;
border-color: rgb(160, 160, 160);
border-radius:10px;
}

QPushButton{
font: 75 10pt &quot;Calibri&quot;;
font-weight: bold;
color: rgb(57, 57, 57);
background-color: rgb(246, 246, 246);
border-style: solid;
border-radius:10px;
width: 80px;
height:24px;
}


QPushButton:hover{
color: rgb(57, 57, 57);
background-color: rgb(246, 246, 246);
border-color: rgb(57, 57, 57);
border-width: 2px;
}

QPushButton:pressed{
color:  rgb(246, 246, 246);
border-color:   rgb(246, 246, 246);
background-color: rgb(57, 57, 57);
}
</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
font: 75 10pt &quot;Calibri&quot;;
color: rgb(57, 57, 57);
background-color: rgb(246, 246, 246);
border-style: groove;
border-color: rgb(160, 160, 160);
border-radius:10px;
}</string>
   </property>
   <property name="title">
    <string>   Dane nadawcy:</string>
   </property>
   <widget class="QLineEdit" name="lekodpoczt_n">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>kod pocztowy</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lenazwisko_n">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>nazwisko</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lemiasto_n">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>miasto</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="letel_n">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>nr telefonu</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leemail_n">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>adres email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lenrdom_n">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>nr. domu</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leimie_n">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>imie</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leulica_n">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>ulica</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lenrlok_n">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>nr. lokalu</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>151</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>   Dane odbiorcy:</string>
   </property>
   <widget class="QLineEdit" name="leimie_o">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>imie</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lenazwisko_o">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>nazwisko</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leemail_o">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>adres email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="letel_o">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>nr telefonu</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leulica_o">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>ulica</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lenrdom_o">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>nr. domu</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lenrlok_o">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>nr. lokalu</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lemiasto_o">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>miasto</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lekodpoczt_o">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>kod pocztowy</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>311</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>   Dane przesy≈Çki:</string>
   </property>
   <widget class="QLineEdit" name="leWaga">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>wymiary [cm]:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leWymiar1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>waga [g]:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leWymiar2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leWymiar3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(93, 93, 93);</string>
    </property>
    <property name="placeholderText">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbZapisz">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>410</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Zapisz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAnuluj">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>410</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Anuluj</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbAnuluj</sender>
   <signal>clicked()</signal>
   <receiver>newdialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
